index.html

```diff
@@ -222,6 +224,72 @@ <h3 class="h3 mt6 mb4">Text Colors</h3>
   <div class="swatch bg-white text-hc-5">text-hc-5</div>
 </div>
 
+<h2 class="h2 mt10 mb4" id="outline">Outline</h2>
+
+<h3 class="h3 mb4">Basic Examples</h3>
+
+<div class="flex">
+  <div class="outline-2-red mr4">div</div>
+  <span class="outline-2-blue mr4">span</span>
+  <div class="outline-2-green mr4" style="border: 1px solid black">border</div>
+  <div class="outline-2-violet" style="border-radius: 4px">border-radius</div>
+</div>
```

Comment:
Would be nice to add code examples below, like this:

![image](https://user-images.githubusercontent.com/2628718/83549623-73afe100-a4ba-11ea-9908-bd5bfa2453d2.png)